### PR TITLE
Add CITATION.cff and update README

### PR DESCRIPTION
**Pull request by murilobalves**
Please check if your names are properly written and if you agree with the order of authors (sorted by number of additions)

### CODE REVIEW

**Review by vellosok75 — APPROVED** [review]

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- family-names: "Resende"
+  given-names: "Ximenes"
+- family-names: "de Sá"
+  given-names: "Fernando Henrique"
+- family-names: "Alves"
+  given-names: "Murilo"
+- family-names: "Ascenção"
+  given-names: "Gabriel"
+- family-names: "Oliveira"
+  given-names: "Ana Clara"
```
Comment (anacso17) [inline CITATION.cff:13]: I don't think this mention is necessary

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Resende"
+  given-names: "Ximenes"
+- family-names: "de Sá"
```
Comment (fernandohds564) [inline CITATION.cff:6]: after the first author, the others must be in alphabetical order.
I think you missed Afonso Mukai and Luana Vilela as authors too.

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

File: CITATION.cff (hunk 1 above)
Comment (murilobalves) [reply CITATION.cff]: ok

File: CITATION.cff (hunk 2 above)
Comment (murilobalves) [reply CITATION.cff:10]: Based on the discussion on https://github.com/lnls-fac/pyaccel/pull/109, we decided to attempt following the order of contribution in the authors.

I adjusted the author's list following the order of pyaccel.

Thanks!

**Review by anacso17 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**